proto/greptime/v1/database.proto

```diff
@@ -13,6 +13,10 @@ service GreptimeDatabase {
   rpc Handle(GreptimeRequest) returns (GreptimeResponse);
 
   rpc HandleRequests(stream GreptimeRequest) returns (GreptimeResponse);
+
+  // HandlePromql service helps to retrieve data via promql or range_promql,
+  // and behaves absolutely the same as Prometheus
+  rpc HandlePromql(PromqlRequest) returns (PromqlResponse);
```

Comment:
This naming looks a bit confusing, but I don't have a better one. Would separating it into another service or something could help?

Reply:
So what's the confusing point?

Reply:
The original idea for having this in a separated service is because its output has a different style with existing apis (raw bytes vs output structure).

Reply:
> The original idea for having this in a separated service is because its output has a different style with existing apis (raw bytes vs output structure).

make sense